backend/external/imagebb.js

```diff
@@ -1,8 +1,22 @@
 const fetch = require('node-fetch');
 const FormData = require('form-data');
 
-const API_KEY = process.env.IMAGE_BB_API_KEY;
-const api = `https://api.imgbb.com/1/upload`;
+const IMGUR = 'imgur'
+const IMGBB = 'imgbb'
+
+var Hosting = IMGBB;
+
+
+var API_KEY = process.env.IMAGE_BB_API_KEY;
+var api = `https://api.imgbb.com/1/upload`;
+var headers = ''
+var url = `${api}?expiration=600&key=${API_KEY}`;
+
+if(Hosting === 'imgur'){
+  API_KEY = process.env.IMGUR_CLIENT_ID
+  url = `https://api.imgur.com/3/image`
+  headers = { "Authorization": `Client-ID ${API_KEY}` }
+}
 
 const uploadImage = async (base64) => {
```

Comment:
Maybe extract it to a separate file  `uploadImage.js`, handle it based on the config variable `Hosting`.
-> use `const` 
-> rename it `IMAGE_HOSTING_PROVIDER`
-> use `switch/case`  instead of `if/else`